network/network.go

```diff
@@ -54,6 +56,14 @@ type Network interface {
 	Server() server.Server
 }
 
+// Message is network message
+type Message struct {
+	// msg is transport message
+	msg *transport.Message
```

Comment:
Should make this exported.

Reply:
Do not export for now since it's not externally useful